src/utils/local/use-toggle.js

```diff
@@ -4,7 +4,7 @@ function useToggle (defaultValue = false) {
   const [value, setValue] = useState(defaultValue)
   const toggle = useCallback((override) => {
     setValue((currentValue) => {
-      return override ?? !currentValue
+      return override === undefined ? !currentValue : !!override
```

Comment:
Any reason we didn't use `Boolean(override)` to do the conversion to Boolean?

Reply:
Oh, well it's because that thought literally never crossed my mind 😆 . Happy to use that approach!